tests/integration/multimodal/yaml/multimodal-concatenate.py

```diff
@@ -8,12 +8,14 @@
 
 
 class ConcatenateMultiModalEncoder(BaseMultiModalEncoder):
+    batch_size = 10
+
     def __init__(self,
                  *args,
                  **kwargs):
         super().__init__(*args, **kwargs)
 
     @batching
     @as_ndarray
-    def encode(self, data: Dict[str, Any], *args, **kwargs):
-        return np.concatenate((data['modality1'], data['modality2']), axis=1)
+    def encode(self, data: 'np.ndarray', *args, **kwargs):
```

Comment:
this needs to be adapted to do concatenation ( I am not so handy with numpy magic)